docs/quickstart.md

```diff
@@ -0,0 +1,195 @@
+# Quick Start
+
+## Welcome to the DevZone of AVSystem!
+
+This Quick Start walks you through the process of building your first LwM2M application using AVSystem’s LwM2M Client **Anjay** and AVSystem’s LwM2M Server **Coiote IoT Device Management**.
+
+Follow this guide to build your first LwM2M application using a simulate device within 20 minutes.
+
+## Access the Coiote IoT DM portal
+
+Get started by <a href="https://eu.iot.avsystem.cloud/" target="_blank">**signing up**</a> to Coiote IoT Device Management. You can sign up for a developer account, allowing you to connect up to 10 devices free of charge.
```

Comment:
`You can sign up for a developer account, which allows you to connect up to 10 devices free of charge.` reads a bit better to me :)